Sources/MessagingPushAPN/MessagingPushAPN.swift

```diff
@@ -76,6 +76,24 @@ public class MessagingPushAPN: MessagingPushAPNInstance {
         messagingPush.trackMetric(deliveryID: deliveryID, event: event, deviceToken: deviceToken)
     }
 
+    /**
+     Configure `MessagingPushAPN`.
+     Call this function in your app if you want to configure the module.
+     */
+    @available(iOSApplicationExtension, unavailable)
+    public static func configure(configOptions configureHandler: ((inout MessagingPushConfigOptions) -> Void)?
```

Comment:
I am leaning towards having this function be inside of `MessagingPush` class instead of in `MessagingPushAPN`. I am assuming the FCM implementation of this feature will need this function therefore, I dont see a reason to also have a separate `MessagingPushFCM.configure` function.

Reply:
Since this is a new public API being added, add an entry to `MessagingPushAPN/APITests` file.

Reply:
Adding this method in MessagingPush might create issues with FCM implementation as Firebase libraries are not accessible in MessagingPush. Also, there is no way to communicate from MessagingPush -> MessagingPushFCM to help passing the control to FCM module.

Hence I am keeping the configure method (name changed to initialize) as is and keeping the common code in MessagingPush.

Reply:
My original comment saying, "have this function inside of `MessagingPush` class" doesn't describe the idea I had well. Sorry about that. 

I see a day where we remove the need for our SDK to have modules initialized. So I am OK with having `MessagingPushAPN.initialize` and `MessagingPushFCM.initialize`.